WFInfo/Ocr.cs

```diff
@@ -804,15 +804,16 @@ internal static void ProcessSnapIt(Bitmap snapItImage, Bitmap fullShot, Point sn
             watch.Start();
             long start = watch.ElapsedMilliseconds;
 
-            string timestamp = DateTime.UtcNow.ToString("yyyy-MM-dd HH-mm-ssff", Main.culture);
+            timestamp = DateTime.UtcNow.ToString("yyyy-MM-dd HH-mm-ssff", Main.culture);
```

Comment:
```suggestion
            string timestamp = DateTime.UtcNow.ToString("yyyy-MM-dd HH-mm-ssff", Main.culture);
```

I think its unsafe to use global variable here.

Reply:
Needed it for [these commented-out lines during development](https://github.com/D1firehail/WFinfo/blob/b45a8684e243efd235e9239dc9d2ecba3f23cd11/WFInfo/Ocr.cs#L1438), should be fine to return it to being a local variable if you paste in a commented-out copy above where it was needed